src/linux/backend-v4l2.cpp

```diff
@@ -786,7 +787,7 @@ namespace librealsense
             // are not available via mipi
             // TODO - find a way to assign unique id for mipi
             // maybe using bus_info and card params (see above in this method)
```

Comment:
Please update the above comment.
and the line comment
Until we will add dynamic binding we use bus info which is unique for a singe deserializer and cam_id which is unique to each camera connected to it, right?

Reply:
done.